src/server/json_family_test.cc

```diff
@@ -668,62 +672,67 @@ TEST_F(JsonFamilyTest, ObjKeys) {
 }
 
 TEST_F(JsonFamilyTest, StrAppend) {
+  map_single_element_vector_ = false;
```

Comment:
should we use this setting to false for all JsonFamilyTest?

Reply:
I am not sure this change map_single_element_vector_ needs to be part of this PR.
We want to check that for json legacy/non legacy mode we return the correct format - i.e array / nester array?
In this case I would create a seprate PR where we set map_single_element_vector_ for all json tests.
In every next PR which adds support to the legacy mode and you add unit tests you can check the relevant format that needs to be returned.
Does this make sense ?